packages/ui-kit-components/src/components/Graph/graph.tsx

```diff
@@ -260,6 +262,15 @@ const Graph: React.FC<GraphProps> = ({
                 borderDash: [5, 2],
                 borderWidth: 1,
               },
+              line2: {
+                type: 'line' as 'box',
+                display: true,
+                yMin: currentData,
+                yMax: currentData,
+                borderColor: palette.success.main,
```

Comment:
The color here should be conditional.  According to the designs it can also be a shade of red.  I'm not sure of the specific situation but I think this may happen when the price is rising and falling.